### DISCUSSION TITLE
판매완료 상태 구현 관련하여

### DISCUSSION THREAD

**Discussion by cho-ja-young** [MajorIn-ABO/MajorIn_FE#37] [Q&A]
현재 채팅이 없는 상태인데, 이때 어떤 식으로 "판매완료"임을 나타내는 API 를 만들면 좋을지

우선 그냥 등록 홈페이지를 하나 만들어서 버튼 하나 만들어놓고 
그 버튼을 누르면 판매 확정이 되고 DB 에 그 내역이 저장되도록 구현해두는게 좋을지.

**Comment by JangGusWjd** [comment]
나는 홈페이지에 판매 확정 버튼을 만들어서 판매자가 그 버튼을 누르면 DB에 그 변경 내용이 저장되어 변경되도록 구현하는게 좋을 것 같아. 판매 완료 버튼은 등록한 판매자만 조회했을 때 나타나게 해야될듯. 판매자와 일반 사용자를 구분할 수 있는 id값 등도 필요할 거 같아

**Comment by wlsl10** [comment]
구매자가 아닌 판매자가 해야 될 것 같고, 버튼 하나를 판매 중과 판매완료로 해서 DB에 저장되도록 구현해두어야  나중에 판매한 글이나 판매 중인 글에 뜰 때 필터링하기 편할 것 같아.
